dags/openshift_nightlies/config/install/hypershift/none-type.json

```diff
@@ -0,0 +1,17 @@
+{
+    "cluster_install_method": "osd",
+    "ocm_cli_version": "master",
+    "ocm_environment": "stage",
+    "rosa_environment": "staging",
+    "rosa_expiration_time": "2880 minutes",
+    "openshift_worker_count": 9,
+    "openshift_network_type": "OpenShiftSDN",
+    "openshift_worker_instance_type": "m5.2xlarge",
```

Comment:
We should default this to the current GA plan which is m6i.4xlarge, however, since m6i instances are scarce go w/ m5.4xl's

Reply:
same comments for the hypershift/ovn.json file as well

Reply:
okay, assuming m6i is available now in all region, but still there may be changes coming as I am not sure if `hosted_cluster_instance_type`(guest worker) should be 4xl or 2xl.